src/pineko/scale_variations.py

```diff
@@ -192,16 +194,21 @@ def compute_ren_sv_grid(grid_path, max_as, nf):
         max as order
     nf : int
         number of active flavors
+    target_path: str
+        path where store the new grid (optional)
     """
     # First let's check if the ren_sv are already there
+    grid_path = pathlib.Path(grid_path)
     grid = pineappl.grid.Grid.read(grid_path)
     sv_as, sv_al = check.contains_ren(grid, max_as, max_al=0)
     if sv_as:
         rich.print(f"[green]Renormalization scale variations are already in the grid")
+        return 0
```

Comment:
Why `0`? `None` is just fine (too much c++? :upside_down_face: )

Reply:
Definetely :)